Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -26,16 +26,43 @@ internal class Program
         static void Main(string[] args)
         {
             Console.OutputEncoding = System.Text.Encoding.Unicode;
+            Dictionary<string, int> playerResult = new Dictionary<string, int> { };
 
             StartIntro();
 
-            int playerResult = StartGame();
+            playerResult = StartGame();
 
             Console.WriteLine("------------------------------------------------------------------------------");
             Console.WriteLine();
             int maxResult = Riddles.Count * SingleGuessPoints * GuessLimit;
-            Console.WriteLine($"Mįslės išspręstos. Surinkti taškai: {playerResult} / {maxResult}");
-            Console.WriteLine();
+            int maxPlayerResult = 0;
+
+            foreach (var playerSum in playerResult)
```

Comment:
instead of foreaching you can use LINQ `Sum` extension and have same logic done in one line, check:https://stackoverflow.com/questions/8128477/how-to-calculate-the-sum-of-all-values-in-a-dictionary-excluding-the-first-item (just you don't need to skip anything, just summ all values)